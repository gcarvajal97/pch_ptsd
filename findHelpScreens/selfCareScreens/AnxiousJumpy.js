import React, { Component } from 'react';
import { View, StyleSheet, Text, ScrollView,} from 'react-native';
import translate from '../../components/translateService';

class AnxiousJumpy extends Component {

  render() {
    return (
        <View>
            <Text style={styles.paragraphTitle}>{translate('selfCare.content.hyperArousal.paragraphTitle1')}</Text>
            <Text style={styles.paragraphBold}>{translate('selfCare.content.hyperArousal.why')}</Text>
            <Text style={styles.paragraph}>{translate('selfCare.content.hyperArousal.paragraph1')}</Text>
            <Text style={styles.paragraphBold}>{translate('selfCare.content.hyperArousal.problem')}</Text>  
            <Text style={styles.paragraph}>{translate('selfCare.content.hyperArousal.paragraph2')}</Text>
            <Text style={styles.paragraphTitle}>{translate('selfCare.content.hyperArousal.paragraphTitle2')}</Text>
            <Text style={styles.paragraphBold}>{translate('selfCare.content.hyperArousal.why')}</Text>
            <Text style={styles.paragraph}>{translate('selfCare.content.hyperArousal.paragraph3')}</Text>
            <Text style={styles.paragraphBold}>{translate('selfCare.content.hyperArousal.problem')}</Text>  
            <Text style={styles.paragraph}>{translate('selfCare.content.hyperArousal.paragraph4')}</Text>
            <Text style={styles.paragraphTitle}>{translate('selfCare.content.hyperArousal.paragraphTitle3')}</Text>
            <Text style={styles.paragraphBold}>{translate('selfCare.content.hyperArousal.why')}</Text>
            <Text style={styles.paragraph}>{translate('selfCare.content.hyperArousal.paragraph5')}</Text>
            <Text style={styles.paragraphBold}>{translate('selfCare.content.hyperArousal.problem')}</Text>  
            <Text style={styles.paragraph}>{translate('selfCare.content.hyperArousal.paragraph6')}</Text>
            <Text style={styles.paragraphTitle}>{translate('selfCare.content.hyperArousal.paragraphTitle4')}</Text>
            <Text style={styles.paragraphBold}>{translate('selfCare.content.hyperArousal.why')}</Text>
            <Text style={styles.paragraph}>{translate('selfCare.content.hyperArousal.paragraph7')}</Text>
            <Text style={styles.paragraphBold}>{translate('selfCare.content.hyperArousal.problem')}</Text>  
            <Text style={styles.paragraph}>{translate('selfCare.content.hyperArousal.paragraph8')}</Text>
            <Text style={styles.paragraphTitle}>{translate('selfCare.content.hyperArousal.paragraphTitle5')}</Text>
            <Text style={styles.paragraphBold}>{translate('selfCare.content.hyperArousal.why')}</Text>
            <Text style={styles.paragraph}>{translate('selfCare.content.hyperArousal.paragraph9')}</Text>
            <Text style={styles.paragraphBold}>{translate('selfCare.content.hyperArousal.problem')}</Text>  
            <Text style={styles.paragraph}>{translate('selfCare.content.hyperArousal.paragraph10')}</Text>
            <Text style={styles.paragraphTitle}>{translate('selfCare.content.hyperArousal.paragraphTitle6')}</Text>
            <Text style={styles.paragraphBold}>{translate('selfCare.content.hyperArousal.why')}</Text>
            <Text style={styles.paragraph}>{translate('selfCare.content.hyperArousal.paragraph11')}</Text> 
            <Text style={styles.paragraphBold}>{translate('selfCare.content.hyperArousal.problem')}</Text>  
            <Text style={styles.paragraph}>{translate('selfCare.content.hyperArousal.paragraph12')}</Text>                
        </View>
    );
  }
}

export default AnxiousJumpy;

const styles = StyleSheet.create({
    paragraphTitle: {
      fontWeight: 'bold',
      fontSize: 20,
      alignSelf: 'stretch',
      textAlign: 'left',
      marginTop: 6,
      marginBottom: 3,
      color: '#2089DC'
    },
    paragraph: {
      textAlign: 'left',
      fontSize: 14,
      marginVertical: 2,
    },
    paragraphBold: {
      textAlign: 'left',
      fontSize: 14,
      fontWeight: 'bold',
      marginVertical: 2,
      marginLeft: 10
    }
  });