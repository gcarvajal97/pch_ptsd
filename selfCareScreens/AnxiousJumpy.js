import React, { Component } from 'react';
import { View, StyleSheet, Text, ScrollView,} from 'react-native';

class AnxiousJumpy extends Component {

  render() {
    return (
            <Text>

                <Text style={styles.title}>
                    HYPER-AROUSAL SYMPTOMS{"\n"}
                </Text>

                <Text style={styles.subTitle}>
                {"\n"}"I find myself worrying a lot that something else bad will happen - or worry a lot about my child being safe."{"\n"}
                </Text>
                <Text style={styles.paragraph}>
                    {"\n"}Why it happens: {"\n"}Being on the lookout for danger makes sense after something scary has happened. It's how we learn to keep ourselves safe from things that are truly dangerous.{"\n"}
                    {"\n"}Is this a problem?: {"\n"}Do you feel this way so strongly, or so much of the time that it is getting in the way of you getting back to normal activities? Does your worry make it harder for you to help your child get back into normal activities?{"\n"}
                </Text>

                <Text style={styles.subTitle}>
                    {"\n"}"I have noticed myself being over-protective of my child since the injury."{"\n"}
                </Text>
                <Text style={styles.paragraph}>
                    {"\n"}Why it happens: {"\n"}All parents want to keep their children safe. An injury can be a harsh reminder that we can't protect our children from every danger 100% of the time. It's normal to feel like tightening up the controls afterwards, and some things may make sense (e.g. paying more attention to safety rules).{"\n"}
                    {"\n"}Is this a problem?: {"\n"}Do you feel this way so strongly, or so much of the time, that you can't let your child get back to normal activities appropriate for his/ her age?{"\n"}
                </Text>

                <Text style={styles.subTitle}>
                {"\n"}"I startle more easily." (e.g., jump at a sudden noise){"\n"}
                </Text>
                <Text style={styles.paragraph}>
                    {"\n"}Why it happens: {"\n"}A frightening experience can trigger an exaggerated physical startle response that takes some time to go away.{"\n"}
                    {"\n"}Is this a problem?: {"\n"}Are you so jumpy that it's really hard to relax? Is it affecting your sleep or ability to concentrate?{"\n"}
                </Text>

                <Text style={styles.subTitle}>
                    {"\n"}"I have been irritable or get angry more easily than before."{"\n"}
                </Text>
                <Text style={styles.paragraph}>
                    {"\n"}Why it happens: {"\n"}Stress and fatigue can lead to feeling irritable. And if you're feeling jumpy or nervous it may be harder to deal with frustrations.{"\n"}
                    {"\n"}Is this a problem?: {"\n"}Is irritability making it hard for you to get along with family or friends? Is this a big change in how you usually deal with stress or frustrations?{"\n"}
                </Text>

                <Text style={styles.subTitle}>
                {"\n"}"I have trouble concentrating."{"\n"}
                </Text>
                <Text style={styles.paragraph}>
                    {"\n"}Why it happens: {"\n"}When we are under a lot of stress, feel really jumpy or on guard, or have distressing thoughts popping into mind, it can be hard to pay attention at times.{"\n"}
                    {"\n"}Is this a problem?: {"\n"}Is difficulty concentrating getting in the way of being able to do your work, make decisions, or take care of your family?{"\n"}
                </Text>

                <Text style={styles.subTitle}>
                    {"\n"}"I have had trouble sleeping."{"\n"}
                </Text>
                <Text style={styles.paragraph}>
                    {"\n"}Why it happens: {"\n"}Sleep problems can be a temporary reaction to disruptions in your life. Feeling jumpy or on guard, or having lots of worried thoughts running through your mind, can make it hard to relax into sleep.{"\n"}
                    {"\n"}Is this a problem?: {"\n"}Are you having a hard time falling asleep or staying asleep many nights? Is this interfering with feeling rested and able to function well during the day?{"\n"}
                </Text>                
                
            </Text>
    );
  }
}

export default AnxiousJumpy;

const styles = StyleSheet.create({
  container: {
      flex: 1,
      alignItems: 'center',
      margin: 10,
      backgroundColor: 'white',
      shadowColor: "#000",
      shadowOffset: {
          width: 0,
          height: 2,
      },
      shadowOpacity: 0.23,
      shadowRadius: 2.62,
      elevation: 4,
  },
  title: {
      backgroundColor: 'white', 
      color: '#2089DC', 
      fontWeight: 'bold', 
      fontSize: 22, 
      textAlign: 'left', 
      alignSelf: 'stretch'
  },
  subTitle: {
    backgroundColor: 'white', 
    color: '#2089DC', 
    alignSelf: 'stretch', 
    paddingLeft: 15, 
    paddingRight:15, 
    fontSize: 15, 
    borderBottomWidth:2, 
    borderBottomColor:'#2089DC',
  },
  paragraph: {
      alignSelf:"center", 
      paddingVertical:1, 
      paddingHorizontal:5,
      fontSize: 14,
      marginTop: 5,
  },
});