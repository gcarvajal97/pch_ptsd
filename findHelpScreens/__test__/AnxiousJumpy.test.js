import React from 'react';
import AnxiousJumpy from '../selfCareScreens/AnxiousJumpy';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
    const tree = renderer.create(<AnxiousJumpy/>).toJSON();
    expect(tree).toMatchSnapshot();
}) 