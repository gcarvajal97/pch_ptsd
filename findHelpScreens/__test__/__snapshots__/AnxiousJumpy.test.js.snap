// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#0077C8",
        "fontSize": 20,
        "fontWeight": "bold",
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    "I find myself worrying a lot that something else bad will happen - or worry a lot about my child being safe."
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "fontWeight": "bold",
        "marginLeft": 10,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Why it happens:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Being on the lookout for danger makes sense after something scary has happened. It's how we learn to keep ourselves safe from things that are truly dangerous.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "fontWeight": "bold",
        "marginLeft": 10,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Is this a problem?:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Do you feel this way so strongly, or so much of the time that it is getting in the way of you getting back to normal activities? Does your worry make it harder for you to help your child get back into normal activities?
  </Text>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#0077C8",
        "fontSize": 20,
        "fontWeight": "bold",
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    "I have noticed myself being over-protective of my child since the injury."
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "fontWeight": "bold",
        "marginLeft": 10,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Why it happens:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    All parents want to keep their children safe. An injury can be a harsh reminder that we can't protect our children from every danger 100% of the time. It's normal to feel like tightening up the controls afterwards, and some things may make sense (e.g. paying more attention to safety rules).
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "fontWeight": "bold",
        "marginLeft": 10,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Is this a problem?:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Do you feel this way so strongly, or so much of the time, that you can't let your child get back to normal activities appropriate for their age?
  </Text>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#0077C8",
        "fontSize": 20,
        "fontWeight": "bold",
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    "I startle more easily." (e.g., jump at a sudden noise)
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "fontWeight": "bold",
        "marginLeft": 10,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Why it happens:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    A frightening experience can trigger an exaggerated physical startle response that takes some time to go away.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "fontWeight": "bold",
        "marginLeft": 10,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Is this a problem?:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Are you so jumpy that it's really hard to relax? Is it affecting your sleep or ability to concentrate?
  </Text>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#0077C8",
        "fontSize": 20,
        "fontWeight": "bold",
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    "I have been irritable or get angry more easily than before."
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "fontWeight": "bold",
        "marginLeft": 10,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Why it happens:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Stress and fatigue can lead to feeling irritable. And if you're feeling jumpy or nervous it may be harder to deal with frustrations.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "fontWeight": "bold",
        "marginLeft": 10,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Is this a problem?:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Is irritability making it hard for you to get along with family or friends? Is this a big change in how you usually deal with stress or frustrations?
  </Text>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#0077C8",
        "fontSize": 20,
        "fontWeight": "bold",
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    "I have trouble concentrating."
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "fontWeight": "bold",
        "marginLeft": 10,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Why it happens:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    When we are under a lot of stress, feel really jumpy or on guard, or have distressing thoughts popping into mind, it can be hard to pay attention at times.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "fontWeight": "bold",
        "marginLeft": 10,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Is this a problem?:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Is difficulty concentrating getting in the way of being able to do your work, make decisions, or take care of your family?
  </Text>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#0077C8",
        "fontSize": 20,
        "fontWeight": "bold",
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    "I have had trouble sleeping."
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "fontWeight": "bold",
        "marginLeft": 10,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Why it happens:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Sleep problems can be a temporary reaction to disruptions in your life. Feeling jumpy or on guard, or having lots of worried thoughts running through your mind, can make it hard to relax into sleep.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "fontWeight": "bold",
        "marginLeft": 10,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Is this a problem?:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Are you having a hard time falling asleep or staying asleep many nights? Is this interfering with feeling rested and able to function well during the day?
  </Text>
</View>
`;
